script/known_failures.txt

```diff
@@ -7,3 +7,73 @@ examples/ruby_spec/core/proc/ruby2_keywords_spec.rb
 examples/ruby_spec/language/fixtures/utf16-be-nobom.rb
 examples/ruby_spec/language/fixtures/utf16-le-nobom.rb
 examples/ruby_spec/language/lambda_spec.rb
+examples/fastlane_fastlane/fastlane/lib/assets/custom_action_template.rb
```

Comment:
why suddently all those new failures?

Reply:
I should remove those. These are from my extended test set.